Bug: omc update does not copy hooks/lib/ directory, causing ERR_MODULE_NOT_FOUND

## Description

After running `omc update` (4.9.3 → 4.10.2), all `.mjs` hook files fail with:

```
node:internal/modules/esm/resolve:275
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/<user>/.claude/hooks/lib/stdin.mjs'
imported from /Users/<user>/.claude/hooks/persistent-mode.mjs
```

## Root Cause

The update process copies `.mjs` hook files to `~/.claude/hooks/` but does **not** copy the `lib/` subdirectory (`stdin.mjs`, `atomic-write.mjs`) from `templates/hooks/lib/`.

The files exist in the plugin cache at:
```
~/.claude/plugins/cache/omc/oh-my-claudecode/4.10.2/templates/hooks/lib/
├── atomic-write.mjs
└── stdin.mjs
```

But `~/.claude/hooks/lib/` is never created during the update.

## Affected Hooks

All `.mjs` hooks that import from `./lib/stdin.mjs`:
- `persistent-mode.mjs`
- `keyword-detector.mjs`
- `pre-tool-use.mjs`
- `post-tool-use.mjs`
- `code-simplifier.mjs`
- `session-start.mjs`

## Steps to Reproduce

1. Have OMC installed via npm (e.g., v4.9.3)
2. Run `omc update`
3. Observe `ERR_MODULE_NOT_FOUND` on every Stop/PreToolUse/PostToolUse hook

## Workaround

```bash
mkdir -p ~/.claude/hooks/lib
cp ~/.claude/plugins/cache/omc/oh-my-claudecode/4.10.2/templates/hooks/lib/*.mjs ~/.claude/hooks/lib/
```

## Expected Behavior

`omc update` (and the installer) should copy `templates/hooks/lib/` alongside the `.mjs` hook files.

## Environment

- OS: macOS (Darwin 24.5.0)
- Node: 22+
- Install method: npm
- OMC version: 4.10.2

--- a/src/__tests__/installer.test.ts
+++ b/src/__tests__/installer.test.ts
@@ -621,4 +621,36 @@ describe('Installer Constants', () => {
       }
     });
   });
+
+  describe('Hook Scripts Installation (#2185 regression)', () => {
+    it('should have all required lib files in templates/hooks/lib', () => {
+      const templatesLibDir = join(getPackageDir(), 'templates', 'hooks', 'lib');
+      expect(existsSync(templatesLibDir)).toBe(true);
+
+      const libFiles = readdirSync(templatesLibDir);
+
+      // Required lib files that must be present
+      const requiredFiles = ['stdin.mjs', 'atomic-write.mjs', 'config-dir.mjs'];
+      for (const file of requiredFiles) {
+        expect(libFiles).toContain(file);
+      }
+    });
+
+    it('should have all standalone hook template files present', () => {
+      const templatesDir = join(getPackageDir(), 'templates', 'hooks');
+      const hookFiles = [
+        'keyword-detector.mjs',
+        'session-start.mjs',
+        'pre-tool-use.mjs',
+        'post-tool-use.mjs',
+        'post-tool-use-failure.mjs',
+        'persistent-mode.mjs',
+        'code-simplifier.mjs',
+      ];
+
+      for (const file of hookFiles) {
+        expect(existsSync(join(templatesDir, file))).toBe(true);
+      }
+    });
+  });
 });

--- a/src/installer/index.ts
+++ b/src/installer/index.ts
@@ -412,18 +412,6 @@ function ensureStandaloneHookScripts(log: (msg: string) => void): void {
     mkdirSync(hooksLibDir, { recursive: true });
   }
 
-  // Copy shared lib modules (stdin.mjs, atomic-write.mjs) required by hook scripts
-  const libSourceDir = join(templatesDir, 'lib');
-  const libTargetDir = join(HOOKS_DIR, 'lib');
-  if (existsSync(libSourceDir)) {
-    if (!existsSync(libTargetDir)) {
-      mkdirSync(libTargetDir, { recursive: true });
-    }
-    for (const libFile of readdirSync(libSourceDir)) {
-      copyFileSync(join(libSourceDir, libFile), join(libTargetDir, libFile));
-    }
-  }
-
   for (const filename of STANDALONE_HOOK_TEMPLATE_FILES) {
     const sourcePath = join(templatesDir, filename);
     const targetPath = join(HOOKS_DIR, filename);

